### DISCUSSION TITLE
'wLE_ISR_ISO-NO_NETWORK.zip' < do you need mecapwn to get this to run ?

### DISCUSSION THREAD

**Discussion by JMaxxL** [israpps/wLaunchELF_ISR#47] [General]
Do I need a modded DVD rom to use the ISO version, I used img burn to make NO_NETWORK img & it would not boot what am I doing wrong...

I would love to have wLaunchELF always ready to run as a backup in drive or even just to mess around with but it would not boot FreeMCBoot just kicked me out when I tried to lauch it

**Comment by israpps** [comment]
> Do I need a modded DVD rom to use the ISO version, I used img burn to make NO_NETWORK img & it would not boot what am I doing wrong...
> 
> I would love to have wLaunchELF always ready to run as a backup in drive or even just to mess around with but it would not boot FreeMCBoot just kicked me out when I tried to lauch it

Hi. to avoid further confusion ISO images are no longer available. they were intended to use in OPL for those who have messy setups and lost access to wLaunchELF. but they seemed to only work for me XD.

Sorry if you wasted a disc
